containers/elabctl/elabctl.sh

```
@@ -0,0 +1,687 @@
+    exit 0
+    ;;
+esac
+
+# default settings that could be overriden by config
+
+declare ELAB_WEB_CONTAINER_NAME='elabftw'
+declare ELAB_MYSQL_CONTAINER_NAME='mysql'
+
+# Now we load the configuration file for custom directories set by user
+if [ -f /etc/elabctl.conf ]; then
+    source /etc/elabctl.conf
```

Shellcheck (0.11.0)

[info] Not following: /etc/elabctl.conf was not specified as input (see shellcheck -x).

(SC1091)